public/js/saveApplicant.js

```diff
@@ -40,6 +41,11 @@ document.querySelector('#submitApplicant').addEventListener('click', e => {
     }
 });
 
+document.querySelector('#saveProgress').addEventListener('click', e => {
```

Comment:
There are actually multiple saveProgress buttons, this only works for the first one. What you'll need to do is change it so the save buttons have a class instead, then use a querySelectorAll to get all of them